nightly/fuzzing.txt

```diff
@@ -1,2 +1,5 @@
-pytest --skip-build --timeout=2h runtime/fuzz_runtime.py
-pytest --skip-build --timeout=2h runtime/fuzz_wasm_vm.py
+pytest --skip-build --timeout=2h fuzz/runtime.py
+pytest --skip-build --timeout=2h fuzz/wasm_vm.py
+pytest --skip-build --timeout=2h fuzz/jsonrpc.py
+pytest --skip-build --timeout=2h fuzz/account_id_serde.py
+pytest --skip-build --timeout=2h fuzz/account_id_borsh.py
```

Comment:
It occurred to me that maybe it’d be better to have a single fuzz test runner and use arguments for each test rather than copying the same script over and over. As in [#5820](https://github.com/near/nearcore/pull/5820).

Reply:
I certainly like yours better. I'll close this in favor of https://github.com/near/nearcore/pull/5820.